docs/source/algorithms/ConvertToConstantL2-v1.rst

```diff
@@ -0,0 +1,32 @@
+.. algorithm::
+
+.. summary::
+
+.. alias::
+
+.. properties::
+
+Description
+-----------
+
+.. warning::
+
+   This algorithm will be deprecated in the next version of Mantid. Please, use :ref:`algm-ConverToConstantL2` instead, which
+   is the new name for this algorithm.
```

Comment:
Should this sentence be in `CorrectFlightPaths-v1` instead of here? Also, I noticed the documentation does not include a user example, would it make sense to add one?